src/strategies/DSL/conditions/priceCondition.ts

```diff
@@ -1,11 +1,51 @@
-import { PrimitiveArgs } from '@brinkninja/types'
-import { PriceConditionArgs } from '@brink-sdk'
+import { PrimitiveArgs, GREATER_THAN_OPERATOR, LESS_THAN_OPERATOR } from '@brinkninja/types'
+import { PriceConditionArgs, UniV3Twap, Token } from '@brink-sdk'
+import { toTokenArgs } from '@brink-sdk/internal/toTokenArgs';
+import { FeeAmount } from '@uniswap/v3-sdk'
 
-function priceCondition (def: PriceConditionArgs): PrimitiveArgs[] {
 
-  // TODO: implement
+const DEFAULT_TIME_INTERVAL = BigInt(1000)
 
-  return []
+function priceCondition ({
+  operator,
+  tokenA,
+  tokenB,
+  price,
+  twapInterval,
+  twapFeePool
+}: PriceConditionArgs): PrimitiveArgs[] {
+  const twap = new UniV3Twap({
+    tokenA: new Token(toTokenArgs(tokenA)),
+    tokenB: new Token(toTokenArgs(tokenB)),
+    interval: twapInterval || DEFAULT_TIME_INTERVAL,
+    fee: twapFeePool as FeeAmount
+  })
+
+  const oracle = {
+    address: twap.address,
+    params: twap.params,
+  }
+
+  switch (operator) {
+    case LESS_THAN_OPERATOR:
+    return [{
+      functionName: 'requireUint256LowerBound', 
+      params: {
+        oracle: oracle,
+        lowerBound: price,
```

Comment:
@phanpak we will need to convert `price` from a number, which potentially has a decimal, to a fixed point 96 integer. There's a function for this in SDK https://github.com/brinktrade/brink-sdk/blob/41c34a0b1bae8d5ad08314a7437de78fdda2af8b/src/oracles/priceToTwapValue.ts. To do this calc, we need to know the number of decimals for both TokenA and TokenB. Simplest way to handle this now is to make the user specify these. So we can just add `tokenA_decimals` and `tokenB_decimals` as params.